daos-cortx/src/daos_kv_benchmark/kv_benchmarking.cc

```diff
@@ -0,0 +1,441 @@
+/**
+ * Benchmarking tests for daos kv store.
+ * Operation type    : put, get, list, remove.
+ * Opeation number   : NR_OPS_XXXX per each opeation
+ */
+#include <benchmark/benchmark.h>
+#include <cstdio>
+#include <daos.h>
+#include <fcntl.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#define POOL_ID "2bd513ad-66d1-4312-829b-d69d8637b455" /* pre-created pool uuid */
```

Comment:
Lets try gflags to make it a param.

Reply:
adopted logic to get pool uuid from environment as using gflags was not possible with existing setup.